deployments/docker/default.env

```diff
@@ -21,4 +21,7 @@ LOG_DIR=/var/log/veraison-logs
 PLUGIN_DIR=/usr/share/veraison/plugins
 
 # Install directory for project input files
-INPUT_FILE_DIR=/usr/share/veraison/input
\ No newline at end of file
+INPUT_FILE_DIR=/usr/share/veraison/input
+
+# Branch/tag to clone the code from
+GIT_CLONE_BRANCH=demo-psa-1.0.1
```

Comment:
I reckon `main` would be a better (i.e., less surprising) default.

(Then the demo docs need to be modified to pass the `GIT_CLONE_BRANCH` override.)